src/rpc/rpc.cc

```diff
@@ -465,6 +465,9 @@ namespace rpc {
                       return FrameType::empty_value();
                   }
                   auto eb = compressor->decompress(std::move(compressed_data));
+                  if (eb.size == 0) {
```

Comment:
What happens if a new code sends empty frame to the old one?

Reply:
The receiver will log an error and drop the connection.

Reply:
It means that when Scylla start to use this feature we need to remember to introduce a feature flag that guards it.

Reply:
We don't need that, because compressors already have their own negotiation mechanism.

If some Scylla version doesn't have the Seastar patch, it also won't have any compressor which uses the empty frames.

I.e. if a new Scylla version includes a new compressor `MY_CUSTOM_COMPRESSOR` which sends empty frames, but it connects to an old Scylla version which doesn't have this compressor, the compression will just be negotiated to `LZ4_FRAGMENTED` and no empty frames will be sent to the old node.

Reply:
Good point. We are not going to extend existing compressors with the feature, but add new once.